egs/swbd/s5c/local/chain/tuning/run_tdnn_7m23b2.sh

```diff
@@ -0,0 +1,501 @@
+#!/bin/bash
+
```

Comment:
Those who are following this PR (esp. @GaofengCheng) might want to have a look at this experiment.
This is a TDNN-only system where the TDNN (view it as a 1x3 convolution) is factorized into two successive  matrices with a bottleneck, both corresponding to (in CNN terminology) 1x2 kernels instead of 1x3; 
and to keep it stable, the first of the two is constrained to be orthonormal.
The comparison below shows that this TDNN-only system is now slightly better than our previous best
system which was a TDNN+LSTM with dropout; and it has just over half the parameters
(23M vs. 39M).
```
# local/chain/compare_wer_general.sh --rt03 tdnn_lstm_1m_ld5_sp tdnn7m23b2_sp
# System                tdnn_lstm_1m_ld5_sp tdnn7m23b2_sp
# WER on train_dev(tg)      12.33     12.38
# WER on train_dev(fg)      11.42     11.44
# WER on eval2000(tg)        15.2      15.1
# WER on eval2000(fg)        13.8      13.6
# WER on rt03(tg)            18.6      18.4
# WER on rt03(fg)            16.3      16.1
# Final train prob         -0.082    -0.084
# Final valid prob         -0.099    -0.098
# Final train prob (xent)        -0.959    -1.049
# Final valid prob (xent)       -1.0305   -1.0661
# Num-parameters               39558436  23120164
```

Reply:
This is amazing!
BTW, have you checked the decoding speed? Is it almost as fast as the baseline TDNN models (i.e. tdnn_7m)?